src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -855,8 +853,7 @@ uint64_t VM_Version::CPUFeatures_parse(uint64_t &glibc_features) {
   return -1;
 }
 
-#if INCLUDE_LD_SO_LIST_DIAGNOSTICS
-
+//#if INCLUDE_LD_SO_LIST_DIAGNOSTICS
```

Comment:
Please remove commented out code.

Reply:
I wanted to indicate this function is used only for `INCLUDE_LD_SO_LIST_DIAGNOSTICS`. I can also put there a full sentence.